src/platforms/hosted/platform.c

```diff
@@ -40,7 +40,13 @@
 
 bmp_info_t info;
 
-jtag_proc_t jtag_proc;
```

Comment:
These changes too feel like they belong with the jtagtap.c/h and dap.c/h changes seperately